utils/featureFlags.ts

```diff
@@ -0,0 +1,33 @@
+import { right } from "fp-ts/lib/Either";
+import { fromEither, TaskEither } from "fp-ts/lib/TaskEither";
+import { InstallationId } from "../generated/notifications/InstallationId";
+import { NHPartitionFeatureFlag } from "./config";
+
+/**
+ *
+ * @param enabledFeatureFlag The feature flag currntly enabled
+ * @param sha the installation id of the user
+ * @returns `true` if the user is enabled for the new feature, `false` otherwise
+ */
+export const getIsInActiveSubset = (
+  isUserATestUser: (InstallationId) => TaskEither<Error, boolean>
```

Comment:
Naming:

we're looking for Beta users, why not call it `isUserABetaUser` or `isBetaUser`?

Reply:
We bounded the concept of `beta` user on the subset of users listed in the table storage, that are very well known.
This function is at higher level, it's meant to tell if the current user is allowed to go through the new step no matter what FF is enabled. For example, in caso of FF=`all` this function will always return `true`.

But I don't like this name neither, so it's fine for me if we give it a better one :D

Reply:
But you were referring to `isUserATestUser`, you're right and actually I changed it in #22  but not here.. my apologies